packages/bruno-electron/src/ipc/network/index.js

```diff
@@ -502,8 +540,25 @@ const registerNetworkIpc = (mainWindow) => {
         });
       }
 
-      const processEnvVars = getProcessEnvVars(collection.uid);
-      interpolateVars(preparedRequest, envVars, collection.collectionVariables, processEnvVars);
```

Comment:
This unfortunately breaks interpolation for the introspection query.
The `runPreRequest` and with that the `interpolateVars` is executed on `request`, but actually executed is `preparedRequest` from https://github.com/usebruno/bruno/pull/705/files#diff-eaeb1da31271859ebecb449a3291ab31d513dbee71aff0df896001827f94cc90R533 in https://github.com/usebruno/bruno/pull/705/files#diff-eaeb1da31271859ebecb449a3291ab31d513dbee71aff0df896001827f94cc90R570.

Not sure how to properly fix this best 🤔

Reply:
Tagging @helloanoop directly, as this should be fixed before the 1.0 release

Reply:
@mheidinger 

I will spend some time next week, This file has gotten really hard to maintain. We have `prepareRequest` and `configureRequest`. Then we have configuring proxy, and the entire request lifecycle along with nuances around running single request, running a folder (collection run), and fetching gql schema.

Need to look at this more holistically. Ideally we should have just 1 separate file with a single`runRequest` method with a good configurability (ex: disable running assertions/tests etc)

For now, to solve this issue, can you update the `fetch-gql-schema` [ipc handler](https://github.com/usebruno/bruno/blob/main/packages/bruno-electron/src/ipc/network/index.js#L551-L626) and duplicate the code inside it needed to run request lifecycle stuff (pre request vars and script, post response vars and script)

@cwilper @mheidinger @grubersjoe If anyone of you have bandwidth, please pick this up.

Its ok to duplicate code.
We'll get to the ideal DRY version post release 1.0.0

Reply:
Created https://github.com/usebruno/bruno/pull/809 to just add back in that one `interpolateVars` call